test/test.js

```diff
@@ -294,8 +285,7 @@ describe('sourcemap-concat', function() {
     });
     builder = new broccoli.Builder(node);
     return builder.build().then(function(result) {
-      expectFile('nothing.js').in(result);
-      expectFile('nothing.map').in(result);
+      expectValidSourcemap('nothing.js').in(result);
```

Comment:
If we are generating a `.map` file, it should always pass validation. Otherwise, downstream parsers may choke on the map--many of them are quite brittle.

An alternative would be to not generate a `.map` file.

The current behavior--generating an invalid `.map`--is not acceptable IMHO.